src/main/java/org/jabref/gui/libraryproperties/LibraryPropertiesViewModel.java

```diff
@@ -31,10 +31,20 @@ public void setValues() {
         }
     }
 
-    public void storeAllSettings() {
```

Comment:
Do not alter public API without discussing it with us. Keep it simple. One method should have only one responsibility. The only job of this method is to store the preferences, not to validate.